app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
```

Comment:
Please, use `()` for importing classes

Reply:
Also, if you want be a good programmer, try to stick to pep8(style guide for python code). In your case, the first import must be `import datetime`, then all related to your project imports